src/main/java/com/puppycrawl/tools/checkstyle/checks/blocks/RightCurlyCheck.java

```diff
@@ -287,16 +286,14 @@ private static boolean isAloneOnLine(Details details) {
     }
 
     /**
-     * Checks whether block has a single-line format.
+     * Checks whether the right curly is on the same line as the next token.
      * @param details for validation.
-     * @return true if block has single-line format.
+     * @return true if right curly is on the same line as the next token.
      */
-    private static boolean isSingleLineBlock(Details details) {
+    private static boolean isRightCurlyOnSameLineAsNextToken(Details details) {
         final DetailAST rcurly = details.rcurly;
-        final DetailAST lcurly = details.lcurly;
         final DetailAST nextToken = details.nextToken;
-        return rcurly.getLineNo() == lcurly.getLineNo()
```

Comment:
This mutation was removed because the inverse is already handled by `isAloneOnLine`.